README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/openpathsampling/openpathsampling.svg?branch=master)](https://travis-ci.org/openpathsampling/openpathsampling)
+![Tests](https://github.com/openpathsampling/openpathsampling/workflows/Tests/badge.svg?branch=master)
```

Comment:
```suggestion
[![Tests](https://github.com/openpathsampling/openpathsampling/workflows/Tests/badge.svg?branch=master)](https://github.com/openpathsampling/openpathsampling/actions?query=workflow%3ATests)
```

(Make it so that it links to the appropriate actions page; I have no idea why GitHub's default badge doesn't do this.)